doc/user/metrics/service-monitor.md

```diff
@@ -0,0 +1,45 @@
+## Using the ServiceMonitor prometheus-operator CRD
+
+[prometheus-operator][prom-operator] is an operator that creates, configures, and manages Prometheus clusters atop Kubernetes.
+
+`ServiceMonitor` is a [CR][cr] of the prometheus-operator, which discovers the `Endpoints` in `Service` objects and configures Prometheus to monitor those `Pod`s. See the prometheus-operator [documention][service-monitor] to learn more about `ServiceMonitor`s.
+
+The `GenerateServiceMonitor` function takes a `Service` object and generates a `ServiceMonitor` resource based on it. To add `Service` target discovery of your created monitoring `Service`s you can use the `metrics.CreateServiceMonitor()` helper function, which accepts the newly created `Service`.
+
+### Prerequisites:
+
+- [prometheus-operator][prom-quickstart] needs to be deployed in the cluster.
+
+### Usage example:
+
+```go
+    import(
+        v1 "k8s.io/api/core/v1"
+        "github.com/operator-framework/operator-sdk/pkg/metrics"
+    )
+
+    func main() {    
+        
+        ...
+
+        // Populate bellow with Service(s) for which you want to create `ServiceMonitor` for.
+        services := []*v1.Service{}
+
+        // Create one `ServiceMonitor` per application per namespace.
+        // Change below value to name of the Namespace you want the `ServiceMonitor` to be created in.
+        ns := "default"
+        
+        // Pass the Service(s) to the helper function, which in turn returns the `ServiceMonitor` object.
```

Comment:
It returns an array of ServiceMonitors right?
https://github.com/operator-framework/operator-sdk/blob/master/pkg/metrics/service-monitor.go#L29 
```suggestion
        // Pass the Service(s) to the helper function, which in turn returns the array of `ServiceMonitor` objects.
```